package.json

```diff
@@ -56,6 +56,7 @@
   "dependencies": {
     "@consento/sync-randombytes": "^1.0.5",
     "@greymass/eosio": "^0.6.4",
-    "cross-fetch": "^3.1.5"
+    "cross-fetch": "^3.1.5",
+    "tsdx": "^0.14.1"
```

Comment:
this should not be a dependancy. if needed it should be  installed globally on the container

Reply:
The `cross-fetch`? I didn't install it. Should I remove it?

Reply:
I've uninstalled tsdx.